reo/validators.py

```diff
@@ -1198,7 +1260,7 @@ def check_special_cases(self, object_name_path, template_values=None, real_value
 
                     # check if user intended to run CHP and supplied sufficient pararmeters to run CHP
                     if user_supplied_chp_inputs:
-                        required_keys = prime_mover_defaults_all['recip_engine'].keys()
+                        required_keys = prime_mover_defaults_all['recip_engine'].keys() # TODO: Check this for NOx rate
```

Comment:
@adfarth is this TODO complete?

Reply:
Yes, this comment is outdated. Removed in [1efd149](https://github.com/NREL/REopt_Lite_API/pull/277/commits/1efd149dafd062deb009b460ccd0984a70ea9e4e)